docs/otel/export-metrics.md

```diff
@@ -16,49 +16,152 @@ instruments not explicitly defined in the specification.
 <!-- toc -->
 
 - [Metric Instruments](#metric-instruments)
-  * [Metric: `otel.processor.spans`](#metric-otelprocessorspans)
-  * [Metric: `otel.exporter.spans`](#metric-otelexporterspans)
+  * [Metric: `otel.processor.items`](#metric-otelprocessoritems)
+  * [Metric: `otel.exporter.items`](#metric-otelexporteritems)
 
 <!-- tocstop -->
 
+## Principles used
+
+This specification defines three levels of detail possible, allowing
+for components to be used with `basic`, `normal`, and `detailed`
+levels that determine which attributes are kept or removed, as by a
+[Metric
+view](https://opentelemetry.io/docs/specs/otel/metrics/sdk/#view).  We
+rely on a conservation principle for pipelines, which states generally
+that what goes in, comes out.
+
+### Signal-independent metric names
+
+OpenTelemetry currently has 3 signal types, but it may add more.
+Instead of using the signal name in the metric names, we opt for a
+general-purpose noun that usefully describes any signal.  The
+signal-agnostic term used here is `items`, referring to spans, log
+records, and metric data points.  An attribute to distinguish the
+`signal` is used, with names designated by the project `traces`,
+`logs`, and `metrics`.
+
+Users are expected to understand that the data item for traces is a
+span, for logs is a record, and for metrics is a point.
+
+### Distinguishing Collectors from SDKs
+
+SDKs and Collectors process the same data in a pipeline, and both
+OpenTelemetry Collector and SDKs are recommended to use the metric
+names specified here.  An attribute to distinguish the `domain` is
+used, with values like `sdk`, `collector`.
+
+In a multi-level collection pipeline, each layer is expected to use a
+unique domain.  This enables calculating aggregates at each level in
+the collection pipeline and comparing them, as a measure of aggregate
+leakage.  Multi-level colletor topologies should allow configuration
+of distinct domains (e.g., `agent` and `gateway`).
+
+### Basic level of detail
```

Comment:
What is the value of having this level? It saves a single binary attribute, but there are plenty of other attributes that are required (domain, name, signal, etc), so the complexity of the spec doesn't seem to be warranted.

Reply:
Saving a boolean attribute means having half as many (i.e., one less) timeseries. The information available in the attribute is almost redundant, so I think having a way to avoid the additional 1 timeseries matters.

When you have metrics on a pipeline, the information available by having a `success` attribute (i.e., one additional series) can be inferred by comparing the subsequent component's totals. This is admittedly a recursive definition -- for the subsequent component to establish it's success/failure rate it will need its own subsequent component's totals, and the final stage in a pipeline will likely not want to use basic-level metrics for this reason. If `Total(x)` is the sum of the single metric for a component X, the recursive rule for deriving Success/Failure of that component is:

```
Dropped(this) = Total(next) - Total(this)
Failed(this) = Dropped(this) + Failed(next)
Success(this) = Total(this) - Failed(this)
```